test/databaseManager.js

```diff
@@ -1,24 +1,70 @@
 import {DataBaseManager} from "../src/database/DatabaseManager.js"
 import chai from 'chai';
 import {Administrator} from "../src/database/modele/Users.js";
+import {Discussion} from "../src/database/modele/Discussion.js";
+import {Question} from "../src/database/modele/Question.js";
 
 const expect = chai.expect;
 const should = chai.should();
 
 describe('Data Base manager test', () => {
     const db = new DataBaseManager();
 
-    before((done) => {
+    before(async () => {
         db.start();
+        let user = null;
+
         const admin = new Administrator({
             login: 'admin',
             password: 'pass'
         });
-        admin.save().then((userSaved) => {
+        await admin.save().then((userSaved) => {
             console.log("administrator saved : ", userSaved);
-            //db.end();
-            done();
+            user = userSaved;
+        });
+
+        const discussion1 = new Discussion({
+            _id: 1,
+            title: "Debate1",
+            description: "My First Debate",
+            startTime: new Date(),
+            administrator: user._id
+        });
+        await discussion1.save().then((discussionSaved) => {
+            console.log("discussion saved : ", discussionSaved);
+        });
+
+        const discussion2 = new Discussion({
+            _id: 2,
+            title: "Debate2",
+            description: "My Second Debate",
+            startTime: new Date(),
+            administrator: user._id
+        });
+        await discussion2.save().then((discussionSaved) => {
+            console.log("discussion saved : ", discussionSaved);
+        });
+
+        const question1 = new Question({
+            _id: 1,
+            titreQuestion: "Question for debate 1",
+            numberVotes: 0,
+            refDiscussion: discussion1._id
+        });
+        await question1.save().then((questionSaved) => {
+            console.log("question saved : ", questionSaved);
+        });
+
+        const question2 = new Question({
+            _id: 2,
+            titreQuestion: "Question 2 for debate 1",
+            numberVotes: 0,
+            refDiscussion: discussion1._id
+        });
+        await question2.save().then((questionSaved) => {
+            console.log("question saved : ", questionSaved);
         });
+
```

Comment:
Empty line -> remove it  later on please

Reply:
Changde with the commit 37bf45f